server/server.ts

```diff
@@ -102,12 +103,12 @@ passport.deserializeUser(User.deserializeUser());
 /* ---------------- */
 /* Server Endpoints */
 /* ---------------- */
-app.use('/dist', [cors(), express.static('dist')]);
-app.use('/static', express.static('static'));
-app.use('/service-worker.js', express.static('static/service-worker.js'));
-app.use('/favicon.png', express.static('static/favicon.png'));
-app.use('/favicon.ico', express.static('static/favicon.png'));
-app.use('/robots.txt', express.static('static/robots.txt'));
+app.use('/dist', [cors(), express.static(path.join(process.cwd(), 'dist/client'))]);
```

Comment:
feels like a tiny helper function to create relative path from `cwd` would be helpful here